src/main/webapp/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
This file does not belong here. We should move contents into .gitignore at the root of repo.